components/form/formitems/overview.md

```diff
@@ -16,6 +16,7 @@ In this article:
 
 * [Basics](#basics)
 * [Example - Customize the appearance of the editors in the Form](#customize-the-appearance-of-the-editors-in-the-form)
```

Comment:
Use the [em dash](https://docs.telerik.com/style-guide/punctuation#lists) instead of a hyphen. 
```suggestion
* [Example—Customizing the appearance of the editors in the Form](#customize-the-appearance-of-the-editors-in-the-form)
```